src/backend/app/tasks/tasks_crud.py

```diff
@@ -50,10 +50,18 @@ async def get_task_count_in_project(db: Session, project_id: int):
 
 def get_task_lists(db: Session, project_id: int):
     """Get a list of tasks for a project."""
-    query = f"""select id from tasks where project_id = {project_id}"""
-    result = db.execute(query)
-    tasks = [task[0] for task in result.fetchall()]
-    return tasks
+    query = text("""
```

Comment:
Do we need the SQL wrapped in `text`? That locks it in a bit more to SQLAlchemy

Reply:
We have used sqlalchemy db session. it was not required in the previous stage. But, it is required since your latest update.  It might be because of pydantic??

Reply:
No it's not related to pydantic, but there is no harm having the `text()` there for now if it works.
We can refactor when we remove SQLAlchemy.